### PR TITLE
MBS-8861: adding text if last login is empty

### PR DESCRIPTION
**Pull request by reosarevok**
I *think* this should work just fine, although I couldn't test it because every user seems to have a last login date in my sandbox.

### CODE REVIEW

File: root/user/profile.tt [hunk 1]
```diff
@@ -91,11 +91,15 @@
             [% END %]
         [% END %]
 
-        [% IF c.user_exists AND (viewing_own_profile OR c.user.is_account_admin);
-             WRAPPER property name=l('Last login:');
-               UserDate.format(user.last_login_date);
-             END;
-           END %]
+        [% IF c.user_exists AND (viewing_own_profile OR c.user.is_account_admin) %]
+           [% WRAPPER property name=l('Last login:') %]
```
Comment (uklauer) [inline root/user/profile.tt]: Why did you change this whole section to go constantly back and forth between TT instructions and pass-through text, cluttering both the source file (with `[% %] [% %]`) and the output (with lots of spaces)?

Comment (reosarevok) [inline root/user/profile.tt]: Because that is how all the other IF-ELSEs are done in this file, so it seemed more consistent.

Comment (uklauer) [inline root/user/profile.tt]: I’d still prefer it as it was before (not surprising, since I intentionally wrote it that way), but if those `[% %]` are necessary, please make them `[%~ ~%]` at least.

**Comment by uklauer** [comment]
Just for the record, the indentation was messed up …
